### DISCUSSION TITLE
[idea]use http in query

### DISCUSSION THREAD

**Discussion by HaveF** [rclement/datasette-dashboards#150] [Ideas]
Hi, this is just a rough idea.

I was thinking that in addition to querying the data in the database to make a dashboard, it would be a great idea to be able to fuse external data sources. Imagine still querying in query, but querying out external data, and then that external data can still be represented as vega, vega-lite, markdown, metric, table, map, isn't it great?

https://simonwillison.net/2022/Aug/10/sqlite-http/
https://observablehq.com/@asg017/introducing-sqlite-http
https://github.com/asg017/sqlite-http

**Comment by HaveF** [comment]
Of course, it seems simpler to store data in SQLite with a different script/program... Well, ignore my idea

**Comment by rclement** [comment]
Well aware of `sqlite-http` and other utilities, this has nothing to do with Datasette Dashboards but only Datasette / SQLite itself.
FYI, you can find an example of using `sqlite-http` in the Datasette Dashboards Demo itself: https://github.com/rclement/datasette-dashboards/blob/14f688cf7d49ead70f2736e7a0a87328e9404e2a/demo/metadata.yml#L148

**Reply by HaveF** [reply]
Thanks! @rclement  I believe I missed it when reading the demo!

**Reply by rclement** [reply]
No worries, there are tons of stuff within the demo to showcase the possibilities 😉
